lib/handlers/patch.js

```diff
@@ -53,30 +53,31 @@ function patchHandler (req, res, next) {
       .then(patchObject => checkPermission(target, req, patchObject))
   ])
   // Patch the graph and write it back to the file
-  .then(([graph, patchObject]) => applyPatch(patchObject, graph, target))
```

Comment:
Not sure, but something may have gone wrong when rebasing.
Because I remember refactoring this file to be fully async, as can be seen here: https://github.com/solid/node-solid-server/blob/release/v5.0.0/lib/handlers/patch.js#L55-L66
So it's possible that there are other changes that are lost as well.

Maybe this can be double-checked?

Reply:
Ah, never mind. That's just because this branch hasn't been rebased yet.